onnxscript/_framework_apis/torch_2_5.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Stable APIs for PyTorch 2.5."""
+
+from __future__ import annotations
+
+__all__ = [
+    "check_model",
+    "convert_version",
+    "get_torchlib_ops",
+    "optimize",
+    "save_model_with_external_data",
+    "OnnxFunctionMeta",
+]
+
+import dataclasses
+import os
+import pathlib
+from typing import Callable
+
+import onnx
+
+from onnxscript import ir
+from onnxscript.ir import _external_data
+
+# Internal flag. Will go away.
+_TORCH_ONNX_SAVE_EXTERNAL_DATA_WITH_IR = (
+    os.getenv("TORCH_ONNX_OFFLOAD_EXTERNAL_DATA_WITH_IR") == "1"
+)
+
+
+@dataclasses.dataclass(frozen=True)
+class OnnxFunctionMeta:
+    """A wrapper of onnx-script function with additional metadata.
+
+    qualified_name: The qualified name of the aten operator.
+    function: The onnx-script function.
+    domain: The domain of the function.
+    name: The name of the function.
+    is_complex: Whether the function is a complex function.
+    """
+
+    qualified_name: str
+    function: Callable
+    domain: str
+    name: str
+    is_complex: bool = False
+
+
+def optimize(model: ir.Model) -> ir.Model:
+    """Optimize the model."""
+
+    # TODO(justinchuby): Use the optimizer
```

Comment:
Just for clarification, what is the purpose of this API. Will it consist of something other than calling optimize.optimize()